internal/api/handlers/fixtures/response/operations_handler/list_operations_history_stage_success.json

```diff
@@ -0,0 +1,25 @@
+{
+  "operations": [
+    {
+      "id": "ae218cc1-2dd8-448b-a78f-0cc979f89f37",
+      "status": "finished",
+      "definitionName": "create_scheduler",
+      "schedulerName": "zooba",
+      "createdAt": "2021-03-01T00:00:00Z"
+    },
+    {
+      "id": "f1fce7b2-3374-464e-9eb4-08b25fa0da54",
+      "status": "finished",
+      "definitionName": "create_scheduler",
+      "schedulerName": "zooba",
+      "createdAt": "2021-02-01T00:00:00Z"
+    },
+    {
+      "id": "c241b467-db15-42ba-b2a8-017c37234237",
+      "status": "finished",
+      "definitionName": "create_scheduler",
+      "schedulerName": "zooba",
+      "createdAt": "2021-01-01T00:00:00Z"
+    }
+  ]
+}
```

Comment:
Two things here.

1. What do you think about renaming this file by changing `history` in favor of `final`?
2. And last but not least and the pagination response fields?